src/Charts/Pie/Pie.js

```diff
@@ -0,0 +1,163 @@
+                select(this)
+                .select('span').style('background-color', pie.color(this.getAttribute('data-id')));
+            })
+            .on('mouseover', function () {
+                pie.focus(this.getAttribute('data-id'));
+            })
+            .on('mouseout', function () {
+                pie.revert();
+            })
+        }
+    }
+
+    _updateChart () {
+        let data = {
+            type: 'pie',
+            columns: this.props.values,
+        };
+
+        let pieConfig = {
+            bindto: '#' + this.props.identifier,
+            data,
+            size: {
+                width: this.props.width,
+                height: this.props.height
+            },
+            legend: {
+                show: false
+            },
+            pie: {
+                width: 8,
+                label: {
+                    show: false
+                }
+            }
+        };
+
+        this.pie = generate(pieConfig);
+
+        this.updateLabels(this.pie);
+    }
+
+    render () {
+
+        const pieClasses = classNames(
+            this.props.className,
+            'ins-c-pie'
+        );
+
+        const wrapperClasses = classNames({
+            'ins-l-pie-wrapper' : true,
+            'legend-right' : this.props.legendPosition === LegendPosition.right,
+            'legend-top' : this.props.legendPosition === LegendPosition.top,
+            'legend-left' : this.props.legendPosition === LegendPosition.left,
+            'legend-bottom' : this.props.legendPosition === LegendPosition.bottom
+        });
+
+        let pieLegend;
+        if (this.props.withLegend) {
+            pieLegend =
+            <div className='ins-l-pie__legend' ref={ref => {this.legend = ref;}}>
+                {this.props.values && this.props.values.map(oneItem => (
+                    <div key={oneItem}  data-id={oneItem[0]} className="pie ins-l-pie__legend--item">
+                        <div className="badge-wrapper">
+                            {/* if this.props.link has a value, wrap the spans in a Link tag */}
+                            <ConditionalLink
+                                key={oneItem[0]}
+                                condition={ this.props.link }
+                                wrap={children =>
+                                    <Link to={`${this.props.link}${oneItem[0].toLowerCase()}`}>
+                                        {children}
+                                    </Link>
+                                }>
+                                <span className="badge"></span>
+                                <span className="badge__label">{oneItem[0]}</span>
+                                <span className="badge__number">({oneItem[1]})</span>
+                            </ConditionalLink>
+                        </div>
+                    </div>
+                ))}
+            </div>;
+        }
+
+        return (
+            <React.Fragment>
```

Comment:
No `React.Fragment` needed here

Reply:
fixed in both Donut and Pie

Reply:
Donut still needs it because it has two sibling tags